classes/03class/exercises/c03-docker05/carineoliveira/c03-docker05-01.txt

```diff
@@ -0,0 +1,53 @@
+Class 03 - Exercise Docker 05
+
+PULL V1
```

Comment:
you don't need to pull the image to run it locally.. when you execute the run command, docker will automatically pull if the image is not stored locally yet